sklearn/mixture/tests/test_gmm.py

```diff
@@ -334,6 +334,73 @@ def test_aic():
         assert_true(np.abs(g.bic(X) - bic) / n_samples < bound)
 
 
+def _test_positive_definite_covars(covariance_type):
+    """ Test that covariance matrices do not become non positive definite
+
+    Due to the accumulation of round-off errors, the computation of the
+    covariance  matrices during the learning phase could lead to non-positive
+    definite covariance matrices. Namely the use of the formula:
+
+    .. math:: C = (\\sum_i w_i \\boldsymbol x_i \\boldsymbol x_i^T)
+              - \\boldsymbol \\mu \\boldsymbol \\mu^T
```

Comment:
Also please remove the `\boldsymbol` markers. They render the expression harder to parse in when reading the source and this latex expression will never be rendered in the HTML or PDF documentation as this is private API anyway.